standard/standard/clause_7_normative_text.adoc

```diff
@@ -397,700 +348,669 @@ The Graph Target supports a network of object relative poses. The graph is a dir
+image::diagram/Stream-Header_SDU.PNG[]
 
 [[stream_trailer_sdu]]
 .Stream Element SDU
-image::diagram/Stream-Element_SDU.PNG[align="center"]
-
-
-|===
-<s|Requirements Class s|Stream 
-
-<s|ID |<<req_stream_sdu,/req/stream/sdu>>
-<s|Target Type|Implemntation of Logical Model
-<s|Description| Stream Logical Model SDU
-<s|Dependency| /req/global
-<s|Requirement S-IHeader-SDU|/req/series_irregular_header/sdu
-<s|Requirement FS-Authority-SDU|/req/frame_specification/authority/sdu 
-<s|Requirement FS-ID-SDU|/req/frame_specification/id/sdu
-<s|Requirement FS-Parameters-SDU|/req/frame_specification/parameters/sdu 
-<s|Requirement S-FT-Element-SDU|/req/irregular_series_frame_and_time/SDU
-<s|Requirement S-Trailer-SDU|/req/regular_series_trailer/sdu
-
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|S-TM-SDU
-<s|ID |<<req_transition_model,/req/transition_model/sdu>>
-2+<|A TransitionModel property SHALL be implemented as one of the values in the TransitionModel enumeration.
-|===
-
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|FS-Authority-SDU
-<s|ID |<<req_sdu_frame_specification_authority,/req/frame_specification/authority/sdu>>
-2+<|The frame_specification.authority attribute SHALL contain a string uniquely specifying a source of reference frame specifications.
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|*Requirement s| FS-ID-SDU 
-<s|ID |<<req_sdu_frame_specification_id,/req/frame_specification/id/sdu>>
-2+<|The frame_specification.ID attribute SHALL be a string uniquely defining a frame within the authority.
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|FS-Parameters-SDU
-<s|ID |<<req_sdu_frame_specification_parameter,/req/frame_specification/parameters/sdu>>
-2+<|The frame_specification.parameter attribute SHALL contain all parameters needed for the corresponding authority and ID.
-|===
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|S-FST-SDU
-<s|ID |<<req_stream_frame_time_element_sdu_,/req/stream/frame_time_element/sdu>>
-<s|Requirement |A FrameAndTimeElement property SHALL be implemented as as an ExplicitFrameSpec and a GeoPoseInstant.
-|===
-
-'''
-
-<<<
-
+image::diagram/Stream-Element_SDU.PNG[]
+
+[[req_stream_sdu]]
+[requirements_class]
+====
+[%metadata]
+label:: /req/stream/sdu
+subject:: Implementation of Logical Model
+description:: Stream Logical Model SDU
+inherit:: /req/global
+
+requirement:: /req/series_irregular_header/sdu
+requirement:: /req/frame_specification/authority/sdu
+requirement:: /req/frame_specification/id/sdu
+requirement:: /req/frame_specification/parameters/sdu
+requirement:: /req/irregular_series_frame_and_time/SDU
+requirement:: /req/regular_series_trailer/sdu
+====
+
+[requirement]
+.S-TM-SDU
+====
+[%metadata]
+label:: /req/transition_model/sdu
+description:: A TransitionModel property SHALL be implemented as one of the values in the TransitionModel enumeration.
+====
+
+[requirement]
+.FS-Authority-SDU
+====
+[%metadata]
+label:: /req/frame_specification/authority/sdu
```

Comment:
This Requirement is duplicated.

Reply:
All duplicated requirements must be removed. Only keep the first instance.